components/uuf-renderablecreator-hbs/src/main/java/org/wso2/carbon/uuf/renderablecreator/hbs/impl/js/LoggerObject.java

```diff
@@ -74,6 +76,23 @@ public void error(Object obj) {
         logger.error(getLogMessage(obj));
     }
 
+    public void error(String message, Object obj) {
```

Comment:
@manuranga: Yes, you can. But as a best practice we advice that the second parameter to be used only to pass an error object. Then the stack trace will be logged along with the error and the custom message.